src/hotspot/share/opto/parse.hpp

```diff
@@ -603,4 +603,55 @@ class Parse : public GraphKit {
 #endif
 };
 
+// Specialized uncommon_trap of unstable_if, we have 2 optimizations for them:
+//   1. suppress trivial Unstable_If traps
```

Comment:
Where is this done?

Reply:
I have another patch based on this one.  I will post it soon.

Reply:
https://github.com/openjdk/jdk/pull/9255

Reply:
hi, @TobiHartmann 
Could you take a look at the new revision?  the 2nd PR is based on the first one.
https://github.com/openjdk/jdk/pull/9255